kernel/kernel-api/src/main/java/io/delta/kernel/internal/util/ColumnMapping.java

```diff
@@ -131,6 +131,14 @@ public static int getColumnId(StructField field) {
     return field.getMetadata().getLong(COLUMN_MAPPING_ID_KEY).intValue();
   }
 
+  public static boolean hasColumnId(StructField field) {
+    return field.getMetadata().contains(COLUMN_MAPPING_ID_KEY);
+  }
+
+  public static boolean hasPhysicalName(StructField field) {
+    return field.getMetadata().contains(COLUMN_MAPPING_PHYSICAL_NAME_KEY);
+  }
```

Comment:
Any reason these need to be made public? Since these are accessors it's fine but don't see anything outside actually using these so it just widens the public surface more than we need to. (Lmk if I'm missing that it's used in another package somewhere or going to be used)

Reply:
I think I'm hallucinating... I distinctly remember asking myself the same question and checking and concluding that they do need to be public... but I cannot find that code anymore. So seems like they can go back to being private haha good catch